recipe/meta.yaml

```diff
@@ -36,6 +36,7 @@ requirements:
     - atomicwrites >=1.0  # [win]
```

Comment:
`tomli >=1.0.0  # [py<311]`, see https://github.com/pytest-dev/pytest/blob/dab199281c8f969ef870d842a660e40872785bd4/setup.cfg#L53

Reply:
`attrs >=19.2.0` isn't required for runtime, it's a test dependency already https://github.com/pytest-dev/pytest/commit/310b67b2271cb05f575054c1cdd2ece2412c89a2
`py >=1.8.2` can be removed, see https://github.com/pytest-dev/pytest/blob/dab199281c8f969ef870d842a660e40872785bd4/setup.cfg#L46-L53
`atomicwrites` is redundant, see https://github.com/pytest-dev/pytest/pull/10115/files